src/fontcache/freetypefontcache.cpp

```diff
@@ -20,14 +20,14 @@
 
 #include "../table/control_codes.h"
 
-#include "../safeguards.h"
-
 #ifdef WITH_FREETYPE
 #include <ft2build.h>
 #include FT_FREETYPE_H
 #include FT_GLYPH_H
 #include FT_TRUETYPE_TABLES_H
 
+#include "../safeguards.h"
```

Comment:
While I guess not really an issue, feels kinda odd to have this inside the ifdef (when none of the others are like that and other headers aren't inside the ifdef either)